### ISSUE TITLE
I can't Dump DBD-EGS???Why?

### ISSUE THREAD

**Issue by ALEHACKsp** [guttir14/UnrealDumper-4.25#96]
I don't know if I'm correct but it seems that this project does not DUMP The DeadByDayLight-EGS-Shipping.exe, in this case, it only dumps STEAM games from EPIC GAMES. I can't use DUMP
I'm trying to dump the SDK but only the folders appear with nothing inside

**Comment by ALEHACKsp** [comment]
![image](https://user-images.githubusercontent.com/45023050/190911124-13395652-6cdc-4f6f-830c-9e1f53618369.png)

**Comment by ALEHACKsp** [comment]
![image](https://user-images.githubusercontent.com/45023050/190911151-7223d28b-fa26-4bb4-a108-aa7a75fa2c9f.png)

**Comment by ALEHACKsp** [comment]
![image](https://user-images.githubusercontent.com/45023050/190911225-57f45636-69e8-4dfa-a7bb-55facba801b1.png)

**Comment by SerseDioRe** [comment]
It dumps also on EPIC Games version, just need to change the sigs

**Comment by ALEHACKsp** [comment]
How to get this news sigs

**Comment by ALEHACKsp** [comment]
> It dumps also on EPIC Games version, just need to change the sigs

Yeah, But how to get sig for this version EpicGames? Please ! , can you help me

**Comment by ALEHACKsp** [comment]
?

**Comment by TempAccountNull** [comment]
> > It dumps also on EPIC Games version, just need to change the sigs
> 
> Yeah, But how to get sig for this version EpicGames? Please ! , can you help me

You have to make it kernel in order to dump this game as you cannot disable EAC. 
EOS is a different story, but even my attempts failed so I just made it a kernel driver and dumped it that way.

Now sigs are easy, just dump the process (kernel) level as you cannot disable the AC or just put the DBD exe in ida, and look for the offsets.

**Comment by ALEHACKsp** [comment]
Thanks Buddy!!!!
